### PR TITLE
AB#629 carnival new styles

### PR DESCRIPTION
**Pull request by franciscoghersi**

### CODE REVIEW

File: src/DonateTo.WebApplication/ClientApp/src/app/app.component.html [hunk 1]
```diff
@@ -1 +1,8 @@
-<router-outlet></router-outlet>
+<nz-layout>
+  <nz-layout>
+    <nz-content>
+      <router-outlet></router-outlet>
+    </nz-content>
+  </nz-layout>
+  <nz-footer style="text-align: center;">© 2020 Softvision. All rights reserved.</nz-footer>
```
Comment (oonosan) [inline src/DonateTo.WebApplication/ClientApp/src/app/app.component.html:7]: Shouldn't we use a translate pipe here?

File: src/DonateTo.WebApplication/ClientApp/src/styles.less [hunk 2]
```diff
@@ -121,6 +141,22 @@ svg:hover {
...
+input:focus,
+input:hover,
+textarea:focus,
+textarea:hover,
+select :focus,
+nz-select-top-control:hover,
+nz-date-picker:hover,
+nz-input-group:hover {
+  border: 0.08em solid #c800a1 !important;
+  border-radius: 6px !important;
```
Comment (oonosan) [inline src/DonateTo.WebApplication/ClientApp/src/styles.less:153]: Use em, please

File: src/DonateTo.WebApplication/ClientApp/src/styles.less [hunk 3]
```diff
@@ -64,6 +64,26 @@ a {
...
+
+// Inputs
+
+input,
+textarea,
+nz-select-top-control,
+nz-date-picker,
+nz-input-group {
+  border: 0.08em solid #c4c4c4 !important;
+  border-radius: 6px !important;
```
Comment (oonosan) [inline src/DonateTo.WebApplication/ClientApp/src/styles.less:80]: Please change px values to em

File: src/DonateTo.WebApplication/ClientApp/src/styles.less (hunk 2 above)
Comment (matigrieben) [inline src/DonateTo.WebApplication/ClientApp/src/styles.less:153]: We don't use px, please change that to em

File: src/DonateTo.WebApplication/ClientApp/src/styles.less (hunk 3 above)
Comment (matigrieben) [inline src/DonateTo.WebApplication/ClientApp/src/styles.less:80]: We don't use px, please change that to em

File: src/DonateTo.WebApplication/ClientApp/src/app/app.component.html (hunk 1 above)
Comment (matigrieben) [reply src/DonateTo.WebApplication/ClientApp/src/app/app.component.html:7]: @Faculewis What do you think?

**Review by matigrieben — CHANGES_REQUESTED** [review]

File: src/DonateTo.WebApplication/ClientApp/src/app/app.component.html (hunk 1 above)
Comment (facundo-lewis) [reply src/DonateTo.WebApplication/ClientApp/src/app/app.component.html:7]: - Remove that message an change it for some text like this:
**For suppport contact support@donate.com**
- It should be in both languages

**Review by matigrieben — APPROVED** [review]

**Pull request merged**